packages/autocomplete/src/containers/AutocompleteContainer.spec.js

```diff
@@ -0,0 +1,582 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { KEY_CODES } from '@zendeskgarden/react-selection';
+import { mountWithTheme } from '@zendeskgarden/react-testing';
+import AutocompleteContainer from './AutocompleteContainer';
+
+/**
+ * Mocks popper.js calls within react-popper due to virtual testing environment
+ */
+jest.mock('popper.js', () => {
```

Comment:
Should this repeated mock be moved into `@zendeskgarden/react-testing`?

Reply:
It looks like the `jest.mock()` call has to occur inline in the component. When imported and ran it doesn't seem to mock correctly.

_I tried this when we originally created the `react-testing` library_